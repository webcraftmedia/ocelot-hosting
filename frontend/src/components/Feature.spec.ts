import { renderSuspended } from '@nuxt/test-utils/runtime'
import Component from './Feature.vue'
import { describe, expect, it } from 'vitest'

describe('Feature', () => {
  it('renders', async () => {
    const html = await (
      await renderSuspended(Component, {
        props: {
          img: 'test.jpg',
          headline: 'Test headline',
          text: 'Test text',
        },
      })
    ).html()
    expect(html).toMatchSnapshot()
  })
})
