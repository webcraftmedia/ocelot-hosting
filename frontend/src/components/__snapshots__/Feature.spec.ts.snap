// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`Feature > renders 1`] = `
"<div id="test-wrapper">
  <div data-v-ac935759="" class="feature"><img data-v-ac935759="" src="test.jpg" alt="">
    <!--v-if-->
    <div data-v-ac935759="" class="content">
      <h3 data-v-ac935759="">Test headline</h3>
      <p data-v-ac935759="">Test text</p>
    </div>
  </div>
</div>"
`;
